### ISSUE TITLE
gem install ufo misses 'render_me_pretty'

### ISSUE THREAD

**Issue by nlinn** [tongueroo/ufo#18]
`$sudo gem install ufo`
Successfully installed ufo-3.0.0
Parsing documentation for ufo-3.0.0
Done installing documentation for ufo after 0 seconds
1 gem installed

`$ ufo init`
/Users/niels/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:120:in require: cannot load such file -- render_me_pretty (LoadError)
	from /Users/niels/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:120:in require
	from /Users/niels/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/ufo-3.0.0/lib/ufo.rb:8:in <top (required)>
	from /Users/niels/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in require
	from /Users/niels/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in require
	from /Users/niels/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/ufo-3.0.0/exe/ufo:11:in <top (required)>
	from /Users/niels/.rbenv/versions/2.4.1/bin/ufo:22:in load
	from /Users/niels/.rbenv/versions/2.4.1/bin/ufo:22:in <main>

After manually installing render_me_pretty everything was fine. Just to let you know.

**Comment by tongueroo** [comment]
@nlinn Thanks so much for the report. Will fix.

**Comment by tongueroo** [comment]
Released v3.0.1 that fixes this: 

```
$ gem install ufo
Fetching: ufo-3.0.1.gem (100%)
Successfully installed ufo-3.0.1
1 gem installed
$ gem which ufo
/Users/tung/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/ufo-3.0.1/lib/ufo.rb
$ ls /Users/tung/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/ufo-3.0.1
CHANGELOG.md    Gemfile         Guardfile       README.md       docs            lib             ufo.gemspec
CONTRIBUTING.md Gemfile.lock    LICENSE.txt     Rakefile        exe             spec            vendor
$ cd !$
cd /Users/tung/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/ufo-3.0.1
$ irb
>> require './lib/ufo'
=> true
>> RenderMePretty
=> RenderMePretty
>> 
$ ```

Thanks again/
